recipes/qt-multimedia/bld.bat

```diff
@@ -0,0 +1,16 @@
+
+cmake -LAH -G "Ninja" ^
+    -DCMAKE_BUILD_TYPE=Release ^
+    -DCMAKE_PREFIX_PATH="%LIBRARY_PREFIX%" ^
+    -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%" ^
+    -DCMAKE_MESSAGE_LOG_LEVEL=STATUS ^
+    -DQT_DEFAULT_MEDIA_BACKEND=ffmpeg ^
+    -B build .
+if errorlevel 1 exit 1
+
+cmake --build build --target install --config Release
+if errorlevel 1 exit 1
+
+:: unversioned exes must avoid clobbering the qt5 packages, but versioned dlls still need to be in PATH
+xcopy /y /s %LIBRARY_PREFIX%\lib\qt6\bin\*.dll %LIBRARY_PREFIX%\bin
+if errorlevel 1 exit 1
```

Comment:
I prefer this because it covers all non-zero exit codes and makes it clear that some magic variable is being checked (same for the other places). Also missing newline at the end.
```suggestion
if %ERRORLEVEL% neq 0 exit 1
```